py/sea/commands.py

```diff
@@ -1632,3 +1674,5 @@ def run(self, args, extra):
 Smc = sea.SeqCmd ('smc', 'alias for fe|opt|smc',
                    [Clang(), Seapp(), SimpleMemoryChecks(), MixedSem(),
                     Seaopt(), Seahorn(solve=True)])
+fpf = sea.SeqCmd('fpf', 'fe|fat-bnd-check|unroll|cut-loops|opt|horn --solve', 
```

Comment:
please rename into `Fpf` to be consistent with other variable names

Reply:
done